### PR TITLE
[m-mr1][upstream version] sony: sepolicy: Address pad_controller denials

### PR DESCRIPTION
**Pull request by humberos**
Required for proximity driver.

Signed-off-by: Humberto Borba <humberos@gmail.com>

### CODE REVIEW

**Comment by humberos** [comment]
Conflicts solved.
Sorry for delay.

File: file_contexts [hunk 1]
```diff
@@ -159,6 +160,17 @@
...
+# AD7146 Proximity
+/sys/devices/virtual/cap_sensor/ad7146/force_calib                  u:object_r:sysfs_force_calib:s0
+/sys/devices/virtual/cap_sensor/ad7146/obj_detect                   u:object_r:sysfs_obj_detect:s0
+/sys/devices/virtual/cap_sensor/ad7146/pad_set                      u:object_r:sysfs_pad_set:s0
+/sys/devices/virtual/cap_sensor/ad7146/pad_num                      u:object_r:sysfs_pad_num:s0
+/sys/devices/virtual/cap_sensor/ad7146/pad_data                     u:object_r:sysfs_pad_data:s0
+/sys/devices/virtual/cap_sensor/ad7146/pad_offset                   u:object_r:sysfs_pad_offset:s0
+/sys/devices/virtual/cap_sensor/ad7146/sw_updata                    u:object_r:sysfs_sw_updata:s0
+/sys/devices/virtual/switch/ad7146_1/state                          u:object_r:sysfs_sw_ad7146_1_state:s0
+/sys/devices/virtual/switch/ad7146_2/state                          u:object_r:sysfs_sw_ad7146_2_state:s0
```
Comment (AdFad666) [inline file_contexts]: Surely all of these can be under one definition, eg `sysfs_pad_controller` it's surely excessive to give a definition for each sysfs when all of them are used by only one service.

Comment (humberos) [inline file_contexts]: Yep, I've just did the same stock way.
But you're right it can be changed.

Comment (AdFad666) [inline file_contexts]: it's done like that upstream? Wow...

Someone was really looking to increase their linecount that day...

**Pull request closed without merging**